core/src/test/kotlin/rest/RestTest.kt

```diff
@@ -86,7 +86,7 @@ class RestServiceTest {
 
         guildId = guild.id
 
-        this@RestServiceTest.guild = kord.getGuild(guildId)!!
+        this@RestServiceTest.guild = kord.getGuildOrNull(guildId)!!
```

Comment:
I think here we should just use `getGuildOrThrow`.

Reply:
Just there or in all applicable situations?

Reply:
in all three places, just was to lazy :D

Reply:
Haha ok

Reply:
All sorted